<launch>
  <node pkg="motor_control" type="marti_odometry_node.py" name="odometry_node" output="screen">
    <rosparam command="load" file="$(find motor_control)/config/motor_param.yaml" />
  </node>
</launch>